library/src/scala/runtime/stdLibPatches/language.scala

```diff
@@ -124,6 +124,7 @@ object language:
      *  @see [[https://github.com/scala/improvement-proposals/pull/84]]
      */
     @compileTimeOnly("`betterMatchTypeExtractors` can only be used at compile time in import statements")
+    @deprecated("The experimental.betterMatchTypeExtractors language import is no longer needed since the feature is now standard", since = "3.6")
```

Comment:
```suggestion
    @deprecated("The experimental.betterMatchTypeExtractors language import is no longer needed since the feature is now standard. Please note that setting an older source version will have no effect of disabling this feature", since = "3.6")
```

Reply:
Good point, and I think it might even clearer to re-state that the setting _never_ has effect. I went with “This setting will have no effect, including when setting an older source version.”

Reply:
I think it is a good message 👍🏻